pkg/apis/kops/networking.go

```diff
@@ -103,7 +103,10 @@ type FlannelNetworkingSpec struct {
 
 // CalicoNetworkingSpec declares that we want Calico networking
 type CalicoNetworkingSpec struct {
-	// BpfEnabled enables the eBPF dataplane mode.
+	// AWSENISrcDstCheck enables/disables ENI source/destination checks (AWS only)
+	// Options: DoNothing (default), Enable, or Disable
+	AWSENISrcDstCheck string `json:"awsSrcDstCheck,omitempty"`
```

Comment:
This name comes from [libcalico-go](https://github.com/projectcalico/libcalico-go/blob/728c26dd8282e7ee95b09ea1f9c5ccee116d814b/lib/apis/v3/felixconfig.go#L355-L357) and I would like to keep it.
I pulled some of the changes related to ordering of fields and comments to https://github.com/kubernetes/kops/pull/10408 to reduce the change list, I hope you don't mind

Reply:
I tend to ignore upstream if it conflicts with kops consistency, but I don't mind if you insist on keeping it. This is cosmetic and is not exposed to users.

Reply:
If you mean the case part, I agree was an oversight. But adding ENI, I am not a big fan.

Reply:
I included "ENI" just because these checks are actually defined on the ENI, not on the instance per se, but then again we didn't name the field "AWSEC2InstanceSrcDstCheck," so my pedantry is not warranted here. I'll change it per your recommendation.